dt: 0.2

our_duckie:
  start_pose: [0.0, 0.0] # [x,y]
  velocity: 1.0
other_duckie:
  start_pose: [0.0, 10.0] # [x,y]
  velocity: 0.25
  type: "slow_duckie" #slow_duckie, unstable_duckie, ...
